runtime/t0rn-parachain/Cargo.toml

```diff
@@ -292,6 +292,8 @@ runtime-benchmarks = [
   "pallet-3vm-ethereum/runtime-benchmarks",
   "pallet-3vm-contracts/runtime-benchmarks",
   "pallet-3vm-account-mapping/runtime-benchmarks",
+  # Other
+  "pallet-celestia-light-client/runtime-benchmarks",
```

Comment:
exclude from benchmarks - breaks compilation for now

Reply:
Disabled in  [90aa16e](https://github.com/t3rn/t3rn/pull/1544/commits/90aa16ef063501d817a0e4830ed91fa6870a0792)